documents/charter.md

```diff
@@ -23,8 +23,8 @@ The Secure Communication of Network Properties (SCONEPRO) Working Group's primar
 
 The properties of this mechanism are as follows:
 
-1. Associativity with an application.
-The network properties must be associated with a given application traversing the network, for example a video playback.
+1. Flow associativity.
+The network communications the properties as they relate to specific (QUIC) flows. This ensures that applications can authorize and apply actions on a per-flow basis.
```

Comment:
The suggested text is headed in the right direction, but 

> communications the properties

is somewhat garbled. I suggested an editorial change to fix that.

More importantly, I THINK I know what

> specific (QUIC) flows

and 

> on a per-flow basis

are getting at, but I'm guessing, because RFC 9000 doesn't use the word "flow" except when describing "flow control" or "IPv6 flow ID", or referring to an example flow. 

Is this about a specific QUIC **connection**? 

If so, I'd suggest this change. 
```suggestion
The network communicates applicable properties as they relate to specific QUIC connections. This ensures that applications can authorize and apply actions on a per-QUIC connection basis.
```